src/osdep/win32/overrides.c

```diff
@@ -371,9 +371,13 @@ win32_ioctl(int fd, long option, int *flag)
 int
 win32_socket(int pf, int type, int protocol)
 {
-	SOCKET s = socket(pf, type, protocol);
+	// SOCKET s = socket(pf, type, protocol);
```

Comment:
What about IPv6?
Did not it work without changes?

Reply:
Didn't test using ipv6. I'll retry with ipv6.